components/layout/HomeLayout.tsx

```diff
@@ -9,14 +11,16 @@ interface Props {
 }
 
 export const HomeLayout: FC<Props> = ({ children, title, pageDescription }) => {
+  const { darkMode } = useContext(UIContext);
+
   return (
-    <div className="h-full">
+    <div className={tw("h-full ", darkMode ? "dark" : "")}>
```

Comment:
does nextjs give you warning of bad hydration due to this?

Reply:
I see, so you've effectively turned off ssr inside the UIProvider. Was that intentional?